drivers/shared/executor/executor_universal_linux.go

```diff
@@ -222,10 +239,11 @@ func (e *UniversalExecutor) configureCG1(cgroup string, command *ExecCommand) {
 		ed = cgroupslib.OpenPath(cpusetPath)
 		_ = ed.Write("cpuset.cpus", cpuSet)
```

Comment:
Is it safe to ignore this error?

Reply:
Historically we had to ignore these errors because people keep running Nomad as non-root or on systems where only some of the cgroup controllers are enabled. But they still want raw_exec to Just Work for them.